api/gateway.proto

```diff
@@ -333,7 +333,7 @@ message GatewayStatus {
     (validate.rules).map = {
       max_pairs: 10,
       keys: { string: { pattern: "^[a-z0-9](?:[_-]?[a-z0-9]){2,}$", max_len: 36 } },
-      values: { string: { max_len: 32 } }
+      values: { string: { max_len: 128 } }
```

Comment:
We never validated this but the values from the TTKG and TTIG are more than 32 chars already. Extending this to pass validation.